decidim-process-extended/app/views/decidim/participatory_processes/admin/participatory_processes/_form.html.erb

```diff
@@ -164,9 +164,11 @@
     <div class="row column">
       <%= form.check_box :private_space, label: t(".private_space") %>
     </div>
+
```

Comment:
Should we edit this file?

Reply:
I got a conflict while doing a merge and after fixing it, I added the spaces for readability.